WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -402,13 +418,10 @@ public class BlockEditorScreen: ScreenObject {
                     .selectAlbum(atIndex: 0)
     }
 
-    private func confirmPost(button: XCUIElement) throws {
-        if FancyAlertComponent.isLoaded() {
-            try FancyAlertComponent().acceptAlert()
-        } else {
-            button.tap()
-            dismissBloggingRemindersAlertIfNeeded()
-        }
+    private func confirmPost(button: XCUIElement, action: postAction) throws {
+        button.tap()
+        guard action == .publish else { return }
+        dismissBloggingRemindersAlertIfNeeded()
```

Comment:
Blogging Reminder is only displayed when a post is published, so we can save the 3 seconds timeout from `dismissBloggingRemindersAlertIfNeeded()` when scheduling a post. ⏱️